src/components/Thanks/ThanksPageSingleVersion.vue

```diff
@@ -159,12 +161,20 @@ const loanForComment = computed(() => {
 	return orderedLoans[0];
 });
 
+const hasPfpLoan = computed(() => props.loans.some(loan => loan?.inPfp));
+
+const hasTeamAttributedPartnerLoan = computed(
+	() => props.loans.some(loan => loan?.distributionModel === 'fieldPartner' && loan?.team?.name)
```

Comment:
I know the ticket said to copy this functionality over, but looking at the code now, I think we probably need to instead check the specific loan, which we have here as `loanForComment`. Otherwise, the first loan might not match what we're checking for.